net/https-dns-proxy/files/https-dns-proxy.init

```diff
@@ -56,19 +46,34 @@ start_instance() {
 	append_parm "$cfg" 'proxy_server' '-t'
 	append_parm "$cfg" 'logfile' '-l'
 	append_bool "$cfg" 'use_http1' '-x'
-	append_match "$cfg" 'verbosity' '-v' '1'
-	append_match "$cfg" 'verbosity' '-vv' '2'
-	append_match "$cfg" 'verbosity' '-vvv' '3'
-	append_match "$cfg" 'verbosity' '-vvvv' '4'
+	append_bool "$cfg" 'ipv4_resolvers' '-4'
```

Comment:
The flag forces either `A` or `AAAA` results for the internal resolver. 
I added an option in case someone wants to use IPv6 and not be forced to use IPv4.
I modified the default config to have this on by default.

Reply:
Still breaks resolution for existing users.

Reply:
I'd implement the use_ipv6_only option instead which if true would remove '-4' from the parameters.

Reply:
Great idea, changed as suggested.

Reply:
README and the init script refer to different variables. ;)
Also, how do you feel about adding `_only` to the variable name? As things are now, a user may want to enable it, thinking it will add IPv6 support, not that it'll remove legacy IP support.

Reply:
Changed as suggested